mineral-ui/mineral-ui
# [Site] update how site is deployed locally

### Description
Updating the local site deploy process to use `netlifyctl` and turning off the auto-publish feature in netlify on `master`.

Netlify is in the process of deprecating the old cli tool, `netlify-cli`, so now we're going to use the new one `netlifyctl`. This also means that I deleted the old `.netlify` file in the root, since everything is going to be looking at `netlify.toml`.

### Motivation and context
Deploying to netlify used to be a little flaky from the command line since you had to remember to
authorize the build in the netlify UI at build time, otherwise you get an auth failure, the build
would fail, and the site would not deploy via the netlify CI. This PR enables deploying directly
from the command line by building and transferring files to netlify hosting. Deploy previews keep
working for all branches including master.

We can always re-hook up auto-deploy when merging to `master` if we choose at some point in the future.

closes #320 

### How to test

1. verify that `package.json` no longer has deploy scripts
2. Log into the netlify UI
3. find the delpoy preview that you would like to publish and click it to view details
4. click "Publish deploy" to publish mineral-ui.com to this deploy preview.

### Types of changes
- scripts (modifies the `npm run deploy` step)

### Checklist



### How does this PR make you feel?

![Alt text here](https://media.giphy.com/media/7ILa7CZLxE0Ew/giphy.gif)
